deploy-service/common/src/main/java/com/pinterest/deployservice/bean/HostBean.java

```diff
@@ -153,9 +89,4 @@ public SetClause genSetClause() {
                     + "last_update=VALUES(last_update),"
                     + "state=VALUES(state),"
                     + "can_retire=VALUES(can_retire)";
-
-    @Override
```

Comment:
As discussed offline, this toString() might be used for non-trivial logic, and it's quite hard to identify all toString() usages since the method can be implicitly called. 
Reverting the change to keep the existing behavior.